plugins/InspectionGadgets/src/inspectionDescriptions/BooleanExpressionMayBeFactorized.html

```diff
@@ -0,0 +1,12 @@
+<html>
+<body>
+Reports any <code>boolean</code> expressions used twice that can be factorized, in order to reduce redundancy and improve the readability.
+Take for example the following expression:
+<pre>a &amp;&amp; b || a &amp;&amp; c;</pre>
```

Comment:
For the 2021.2 release we have standardized the format of the example part of the description. It looks like this:
```
<p><b>Example:</b></p>
<pre><code>
  <!-- example here -->
</code></pre>
<p>After the quick-fix is applied:</p>
<pre><code>
  <!-- fixed example here -->
</code></pre>
```
Also you could remove the semicolons. They are not relevant for the example and the statement is not compilable anyway.